package.json

```
@@ -95,7 +95,16 @@
     "cypress:run": "cypress run",
     "lint": "eslint src/**/*.{js,jsx,ts,tsx}",
     "lint:fix": "eslint --fix src/**/*.{js,jsx,ts,tsx}",
-    "format": "prettier --write \"./src/**/*.{js,jsx,ts,tsx,css,md}\" --config ./.prettierrc"
+    "format": "prettier --write \"./src/**/*.{js,jsx,ts,tsx,css,md}\" --config ./.prettierrc",
+    "prepare": "husky install",
+    "pre-commit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.+(js|jsx|ts|tsx|json|css|md)": [
```

```suggestion
    "*.+(js|jsx|ts|tsx)": [
```
Eslint is currently configured to only test and fix for JavaScript and TypeScript files.
We can add corresponding plugins to extend this feature within `.eslintrc.js` file.